### ISSUE TITLE
Converted folders are empty?

### ISSUE THREAD

**Issue by shimizurei** [dazedcat19/FMD2#215]
```
https://github.com/dazedcat19/FMD2/releases/tag/2.0.2.0

Download the converter tool win32 or win64 for your system and the modules.txt file.
Extract it and place converter.exe and modules.txt in your old FMD folder.
Run the converter tool, after a few moment there will be a folder called converted.
Copy the contents of that folder to your new FMD folder.
```

I followed the aforementioned instructions and my data and userdata folders were empty. Modules.txt worked though. Am I missing something? I just want to move my FMD1 databases over...

Are the database structures the same? Can I just move my favorites, downloadedchapters, and downloads dbs to FMD2's userdata folder?

According to another [post](https://github.com/dazedcat19/FMD2/issues/53#issuecomment-656473462), this is what is supposed to be in the other folders:

```
settings.json (main fmd settings)
downloads.db (download list)
favorites.db (favorite list)
downloadedchapters.db (downloaded chapters history)
```

I get nothing. I've closed both programs and run the converter multiple times.

---

FMD1 version: fmd_1.3.2.0_x86_64-win64.7z
FMD2 version: fmd_2.0.30.0_x86_64-win64.7z
Converter version used: converter_x86_64-win64.7z (+modules.txt)

**Comment by Slasar41** [comment]
I think you downloaded the old version of converter tool.
You should get it here: https://github.com/dazedcat19/FMD2/releases/tag/2.0.21.0

**Comment by shimizurei** [comment]
It took some trial and error but I got it figured out. I had extracted the converter file to it's own folder out of habit. The `converter.exe` file needs to be with the other exe files and folders. Then the `converted` folder will contain the folders `userdata` and `data`. I took those folders out and overwrote the old folders (of the same names) with my versions (I had previously taken the whole `converted` folder and copied it to my FMD2 directory). Then I opened it up and I saw my list. I'm dumb, so the instructions were kind of unclear to me ^_^;

There's a more recent converter file here: https://github.com/dazedcat19/FMD2/releases/tag/2.0.22.0
